### PR TITLE
Reload on file change

### PR DESCRIPTION
**Pull request by hholierhoek**
Part 2 - the file watcher. README included this time.

I also switched to a consistent use of loggerContext instead of context to avoid the cast, since we're passing it around everywhere anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +72,7 @@ The general configuration format is as follows:
 
 ```scala
 logback {
+  scanPeriod = 30 seconds
```
Comment (satabin) [inline README.md]: It seems I can never be completely happy :smile:, but could you use dashes instead of camel case (for consistency reasons)

Comment (satabin) [inline README.md]:
```suggestion
  scan-period = 30 seconds
```

Comment (hholierhoek) [reply README.md]: No problem. I like consistency too - just not that many multi-word things in there so I missed that.

**Comment by hholierhoek** [comment]
Sorry, search fail!

**Comment by satabin** [comment]
version `0.3.0-SNAPSHOT` published with that change.

**Pull request merged**